src/main/webapp/app/shared/date-time-picker/date-time-picker.component.ts

```diff
@@ -10,7 +10,16 @@ import { Moment, isMoment } from 'moment';
             {{ labelName }}
         </label>
         <div class="d-flex">
-            <input class="form-control position-relative pl-5" [ngModel]="value" [disabled]="disabled" (ngModelChange)="updateField($event)" [owlDateTime]="dt" name="datePicker" />
+            <input
+                class="form-control position-relative pl-5"
+                [ngClass]="{ 'is-invalid': error }"
+                [ngModel]="value"
+                [disabled]="disabled"
+                [min]="this.getCurrentDate()"
```

Comment:
could this be a problem if we change the date after it has been due?

Reply:
@krusche I will have a look at it

Reply:
@krusche Hi Stephan, I removed the calendar restriction so there shouldn't be a problem when editing the dates.